controllers/update_status_test.go

```diff
@@ -318,7 +313,7 @@ var _ = Describe("update_status", func() {
 
 		When("process group has no Pod", func() {
 			It("should be added to the failing Pods", func() {
-				processGroupStatus := fdbv1beta2.NewProcessGroupStatus("storage-1337", fdbv1beta2.ProcessClassStorage, []string{"1.1.1.1"})
+				processGroupStatus := fdbv1beta2.NewProcessGroupStatus("storage-100000", fdbv1beta2.ProcessClassStorage, []string{"1.1.1.1"})
```

Comment:
I don't follow the reasoning for this change

Reply:
We add a new process group here and the idea was to just add a process group with the id `100000` as `99999` is the maximum generated id. The idea was to make sure we prevent conflicts where the randomness chooses the ID 1337. I updated the code with another solution I used in a different place.

Reply:
oooh, neat! that makes sense

Reply:
I decided just to delete the pod in that test case, not sure why we didn't do that before 🤔